src/custom/state/lists/hooks/hooksMod.ts

```diff
@@ -1,6 +1,119 @@
+import { useMemo } from 'react'
 import { useSelector } from 'react-redux'
 import { AppState } from '../..'
+import { ChainId } from '@uniswap/sdk'
+import { useActiveWeb3React } from 'hooks'
+import { DEFAULT_NETWORK_FOR_LISTS, UNSUPPORTED_LIST_URLS } from 'constants/lists'
+import { TokenList } from '@uniswap/token-lists'
+import DEFAULT_TOKEN_LIST from '@uniswap/default-token-list'
+import { TokenAddressMap, listToTokenMap, combineMaps, EMPTY_LIST } from '@src/state/lists/hooks'
+import sortByListPriority from 'utils/listSort'
+import UNSUPPORTED_TOKEN_LIST from 'constants/tokenLists/uniswap-v2-unsupported.tokenlist.json'
 
-export function useFullLists() {
-  return useSelector<AppState, AppState['lists']>(state => state.lists)
+export * from '@src/state/lists/hooks'
+
+// MOD
```

Comment:
You deleted here lots of lines of this mod:
![image](https://user-images.githubusercontent.com/2352112/111204421-62a60b80-85c6-11eb-8322-fa9543af1758.png)

You can leave the lines, and comment them if you don't need them

Reply:
is it optimal to do this when I'm not deleting them but simply importing them from the base file? regardless i copied them back in and left them commented out

Reply:
The rule we don't want to breake is:
`The mod always shoud compare line by line.` so we don't delete

I guess you refer to:
```
export * from '@src/state/lists/hooks'
```

That line above should be used in the `index.ts`, and then, the things you don't use in the Mod, should be there but commented out. 

Also, 
![image](https://user-images.githubusercontent.com/2352112/111304055-8b291680-8655-11eb-81f6-d38b6c33d78e.png)

Reply:
ok the changes i made address this

Reply:
https://github.com/gnosis/gp-swap-ui/pull/244/commits/4b83cbbb73353d9f672e159dd3225fa2bd2e1589